test/ui-test/testSuites/global_shared/steps/steps.py

```diff
@@ -88,9 +88,9 @@ def step(context, obj):
 ###########################################################################
 
 def the_user_restarts_the_app(context: any):
-    waitFor(lambda: currentApplicationContext().detach(), 500)
+    init_steps.waitFor(lambda: init_steps.currentApplicationContext().detach(), 500)
```

Comment:
I think these two methods are not part of the `init_steps` file.
```suggestion
   waitFor(lambda:currentApplicationContext().detach(), 500)
```